RecoTracker/TkSeedingLayers/src/HitExtractorSTRP.cc

```diff
@@ -208,12 +236,25 @@ HitExtractor::Hits HitExtractorSTRP::hits(const TkTransientTrackingRecHitBuilder
       if (skipClusters)
         cleanedOfClusters(ttrhBuilder, ev, result, false, cleanFrom);
     }
+    if (hasVectorHits) {
+      LogError("HitExtractorSTRP") << "TIB is not supposed to be in Phase2 TRK detector configuration. What follows "
+                                      "have never been checked before! ";
+      edm::Handle<VectorHitCollectionNew> vectorHits;
+      ev.getByToken(theVectorHits, vectorHits);
```

Comment:
```suggestion
      auto const& vectorHits = ev.get(theVectorHits);
```
handle is not needed, get the product directly

Reply:
Done.